macros.in

```diff
@@ -145,6 +145,9 @@
 #%__systemd_sysusers	@__SYSTEMD_SYSUSERS@
 %__systemd_sysusers	%{_rpmconfigdir}/sysusers.sh
 
+# enable debug package generation
+%_enable_debug_packages 1
```

Comment:
I realized this doesn't belong to the main macros.in, really - it should probably be in platform.in instead. For one, this is a Linux-only feature due to the ties elfutils, but also: we don't want debug packages for noarch packages, *that* is the place to express it. Sadly we'll still need the other noarch checks because the platform files aren't reread on spec parse recursion, but it's the right direction anyhow.